### ISSUE TITLE
Home page image handling

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#343]
- icons for Resource type should be font icons rather than images (@jvine to provide list)
- for other images (Featured sets, Articles), set alt="" and tabindex=-1 (minus one)
